transcendence-app/src/chat/chat.controller.ts

```diff
@@ -96,4 +96,20 @@ export class ChatController {
     }
     return chatRooms;
   }
+
+  @Get('room/:id')
+  @ApiCreatedResponse({
+    description: 'get a chatroom',
+    type: ChatRoom,
+  })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable entity' })
+  async getChatRoomById(@Param('id', ParseUUIDPipe) chatRoomId: string) {
+    const chatroom = await this.chatService.getChatRoomById(chatRoomId);
+
+    if (!chatroom) {
+      throw new UnprocessableEntityException();
```

Comment:
suggestion: Here we could use the `NotFoundException()`

Reply:
Done